crates/types/src/config/common.rs

```diff
@@ -603,40 +624,38 @@ pub enum MetadataStoreClient {
 )]
 // TODO(azmy): Remove this Shadow struct once we no longer support
```

Comment:
What's preventing us from cleaning this up now?

Reply:
Right now we still need it for backward compatibility.